packages/core/src/components/Checkbox/CheckboxRow.tsx

```diff
@@ -184,5 +195,23 @@ export const SEED_DATA = {
       valuePropName: "status",
       handlerPropName: "onPress",
     }),
+    color: createColorProp({
+      group: GROUPS.basic,
```

Comment:
you can remove group, label, defaultValue from these - the defaults you set are the defaults of the function